projects/00-trivia/src/main.js

```diff
@@ -0,0 +1,3 @@
+/*
+  Aquí van tus funciones
```

Comment:
Este comentario me parece un poco confuso... en este archivo va el código JavaScript, pero no solo funciones... no?
```suggestion
  Aquí va tu JavaScript
```